src/components/Home.js

```diff
@@ -57,9 +55,11 @@ const Home = () => {
           onChange={(e) => setSharedToken(e.target.value)}
         />
         {sharedTokenError && <p>Token is not found. Please try again.</p>}
-        <Button type="submit" variant="contained" color="primary">
-          Join an existing list
-        </Button>
+        <GreenButton
+          type="submit"
+          clickFunction={verifyToken}
+          btnText="Join an existing list"
+        />
```

Comment:
I added a `clickFunction` prop here with the `verifyToken` function passed in. I didn't have this before, and before using the Material-UI button component, I don't think we needed to have a handleClick for this since it was part of the form. This _should_ solve the problem where the user wasn't able to sign in with an existing token if they clicked the button.

Reply:
This works too! 👏🏻🎉